cmd/sonobuoy/app/results.go

```diff
@@ -235,6 +259,17 @@ func printSinglePlugin(input resultsInput, r *results.Reader) error {
 		}
 		_, err = io.Copy(os.Stdout, fReader)
 		return err
+	} else if input.mode == resultModeReadable {
+		fReader, err := r.PluginResultsReader(input.plugin)
+		if err != nil {
+			return errors.Wrapf(err, "failed to get results reader for plugin %v", input.plugin)
+		}
+		writer := &humanReadableWriter{os.Stdout}
+		_, err = io.Copy(writer, fReader)
+		if err != nil {
+			panic(err.Error())
```

Comment:
Panic is good for quick POC and iteration, but you'll want to return the error wrapped with some context.

```
return errors.Wrapf(err, "failed to copy data for plugin %v", input.plugin")
```